LearnositySDK/Request/Init.cs

```diff
@@ -422,7 +428,7 @@ private void setServiceOptions()
                         signatureList.Add(this.securityPacket.getString("user_id"));
                         signatureList.Add(this.secret);
 
-                        string signature = this.hashValue(signatureList.ToArray());
```

Comment:
I see the signatureList is only here, so maybe the declaration should be removed too?